src/generic_core/remote-connection.ts

```diff
@@ -296,12 +297,20 @@ var generateProxyingSessionId_ = (): string => {
           pc = bridge.preObfuscation('sockstortc', config, this.portControl_);
           break;
         case 3:
-          log.debug('using bridge with basicObfuscation');
+          log.debug('using bridge with caesar obfuscation');
           pc = bridge.basicObfuscation('sockstortc', config, this.portControl_);
           break;
+        case 4:
```

Comment:
Some people advise adding a comment like `// fall through to case BAZ` when falling through is intended just to make it super clear it's not an oversight, but totes up to you. (I'm not one of those zealots who argues that switch/case statements are too error-prone to even ever use, but I can see where they're coming from.)